src/Core/Binary.idr

```diff
@@ -122,8 +122,8 @@ HasNames e => HasNames (TTCFile e) where
           = pure $ Just $ MkRewriteNs !(full gam e) !(full gam r)
 
       fullPrim : Context -> PrimNames -> Core PrimNames
-      fullPrim gam (MkPrimNs mi ms mc)
-          = pure $ MkPrimNs !(full gam mi) !(full gam ms) !(full gam mc)
+      fullPrim gam (MkPrimNs mi ms mc md)
+          = pure $ MkPrimNs !(full gam mi) !(full gam ms) !(full gam mc) !(full gam md)
```

Comment:
Why not `[| MkPrimNs (full gam mi) (full gam ms) (full gam mc) (full gam md) |]` instead?

Reply:
Thanks, buzden. Rebased and fixed this a part of other refactorings.